style/amenity-points.mss

```diff
@@ -1471,7 +1471,8 @@
   }
 }
 
-#amenity-low-priority {
+#amenity-low-priority,
+#railway-crossings {
```

Comment:
Since there is no overlap in style rules between the `amenity-low-priority` and `railway-crossings` layers the style rules for those should be separate.